### PR TITLE
sql: update content structure and wording

### PR DESCRIPTION
**Pull request by CaitinChen**
@tiancaiamao @lilin90 PTAL

### CODE REVIEW

File: sql/privilege.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ category: user guide
 
 # 权限管理
 
-## 权限管理概述
+TiDB 的权限管理系统是按照 MySQL 的权限管理进行实现，TiDB 支持大部分的 MySQL 的语法和权限类型。
```
Comment (lilin90) [inline sql/privilege.md]:
```suggestion
TiDB 的权限管理系统按照 MySQL 的权限管理进行实现，TiDB 支持大部分的 MySQL 的语法和权限类型。
```

File: sql/privilege.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ mysql> SELECT user,host FROM mysql.user WHERE user='xxxx';
 1 row in set (0.00 sec)
 ```
 
-例子中 `xxxx@%` 就是自动添加进去的用户。
+上述示例中，`xxxx@%` 就是自动添加的用户。
```
Comment (lilin90) [inline sql/privilege.md]:
```suggestion
上述示例中，`xxxx@%` 即自动添加的用户。
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL again.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**